data/migrations/20190321220524_createIngredientsTable.js

```diff
@@ -0,0 +1,22 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('ingredients', tbl => {
+    tbl.increments();
+    tbl.string('ingredient')
+       .notNullable();
+
+    tbl.float('qty')
```

Comment:
What happens if you need a different quantity for the same ingredient? Really, you should have another table linking ingredients to recipes (with their respective quantities).